src/controllers/chatController.ts

```diff
@@ -21,6 +21,20 @@ export const chatController = {
     const gameIndex: number = util.gameToIndex(internalGameName);
     res.send(await getChatFromDB(gameIndex));
   },
+  /**
+   * Get all chat messages for the game_name specified.
+   * Sends the results as JSON.
+   * @param req Request object
+   * @param res Response object
+   */
+  async postToChat(req: express.Request, res: express.Response) {
+    const internalGameName: string = req.params['game_name'];
```

Comment:
Do you need "internal" in this name? It doesn't seem like there's any other "game name" in this function.

Reply:
Done https://github.com/knative-portability/Kubercade/pull/20/commits/7f706ae8393b9d7d08195a8f38616a4edece0d4e. It was originally copied from `scoresController.ts` which uses both `internalGameName` (pac-man) and `prettyGameName` (Pac-Man).